extensions/iceberg/src/main/java/io/deephaven/iceberg/util/TableWriterOptions.java

```diff
@@ -61,6 +61,20 @@ Map<String, Integer> dhColumnNameToFieldId() {
         return reversedMap;
     }
 
+    /**
+     * Used for providing {@link org.apache.iceberg.SortOrder} to sort new data while writing to an iceberg table using
+     * this writer.
+     * <p>
+     * Users can specify how to extract the schema in multiple ways (like disable sorting, use table default, by ID).
+     * <p>
+     * Defaults to {@link SortOrderProvider#useTableDefault()}, which means sort new data using the table's default sort
+     * order.
```

Comment:
I don't usually like duplicating documentation; ie, just link to `useTableDefault`, but I don't think we need to explain what it is assuming `useTableDefault` has the proper javadoc.